pkg/commands/kustomizationfile_test.go

```diff
@@ -158,7 +160,7 @@ vars:
     kind: Service
     name: my-service
 
-bases:
+BASES:
```

Comment:
so this just tests case insensitivity?  nice.

Reply:
yes